oauth2client/client.py

```diff
@@ -1872,6 +1878,7 @@ def __init__(self, client_id,
         self.device_uri = device_uri
         self.token_info_uri = token_info_uri
         self.authorization_header = authorization_header
+        self.pkce = pkce
```

Comment:
Consistency? All the other instance fields are "public". (And after all, there's really no such thing as a private attribute in Python)